packages/testing/src/deployment.cairo

```diff
@@ -38,13 +46,19 @@ pub fn deploy_another_at(
 }
 
 /// Combines the declaration of a class and the deployment of a contract into one function call.
+/// This function will skip declaration if the contract is
+/// already declared(the result of `snforge_std::declare` call is of type
+/// `DeclareResult::AlreadyDeclared`)
 pub fn declare_and_deploy(contract_name: ByteArray, calldata: Array<felt252>) -> ContractAddress {
     let contract_class = declare_class(contract_name);
     deploy(contract_class, calldata)
 }
 
 /// Combines the declaration of a class and the deployment of a contract at the given address
 /// into one function call.
+/// This function will skip declaration if the contract is
+/// already declared(the result of `snforge_std::declare` call is of type
```

Comment:
```suggestion
/// already declared (the result of `snforge_std::declare` call is of type
```

Reply:
I wonder if we can add this to the typos workflow or to the fmt action